modules/install/wizard.php

```diff
@@ -27,25 +27,24 @@
             $row = $database->queryWithOnlyFirstRow("SELECT email FROM %prefix%user WHERE email = ?", [$_POST["email"]]);
 
             // If found, update password
-            if ($row !== false) {
-                $db->qry(
-                    "UPDATE %prefix%user SET password = %string%, type = '3' WHERE email=%string%",
-                    md5($_POST["password"]),
-                    $_POST["email"]
-                );
-            // If not found, insert
-            } else {
-                $db->qry(
-                    "INSERT INTO %prefix%user SET username = 'ADMIN', firstname = 'ADMIN', name = 'ADMIN', email=%string%, password = %string%, type = '3', lastlogin = NOW(), comment = '', birthday = NULL, signature = ''",
-                    $_POST["email"],
-                    md5($_POST["password"])
-                );
-                $userid = $db->insert_id();
```

Comment:
Are we 100% sure that `$userid` is not used in an include context?
Because this is not retrieved in the new code.

Reply:
Hi Andy,

I had the same thought, but there was no other occurrence in the file and multiple end-to-end runs of the installer showed no warnings or issues.

My guess is that this was an leftover from when login functionality was transferred to `Auth->login`